src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.html

```diff
@@ -14,8 +14,15 @@
 
 <!-- Completion Page -->
 <div *ngIf='isFinished()' class='roll-call-complete' [@completeRollCall]>
-  <span class='fa fa-check-circle-o roll-call-complete-icon'></span><br>
-  Roll call completed.
+  <span class='fa fa-check-circle-o roll-call-complete-icon'></span>
+  <div>
+    Roll call completed.
+  </div>
+  <div>
+    <button mat-raised-button color="primary" class="roll-call-complete-save nav-button" (click)="finish()" fxFlex>
+      <b>Save</b>
```

Comment:
Is there a reason why it is not immediately saved, once the roll call is finished?

Reply:
That's a good idea!